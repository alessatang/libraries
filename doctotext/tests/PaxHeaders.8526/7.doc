30 atime=1389151475.242878847
30 ctime=1389151475.242878847
