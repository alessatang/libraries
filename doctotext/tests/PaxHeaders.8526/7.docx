30 atime=1389151475.258878781
30 ctime=1389151475.258878781
