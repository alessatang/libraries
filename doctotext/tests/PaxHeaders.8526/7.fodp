30 mtime=1384978086.645497406
30 atime=1389151475.347878414
30 ctime=1389151475.347878414
