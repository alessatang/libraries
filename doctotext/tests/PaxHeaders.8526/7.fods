30 mtime=1384978086.646497406
29 atime=1389151475.34887841
29 ctime=1389151475.34887841
