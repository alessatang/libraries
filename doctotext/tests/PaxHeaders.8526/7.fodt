30 mtime=1384978086.648497407
30 atime=1389151475.363878348
30 ctime=1389151475.363878348
