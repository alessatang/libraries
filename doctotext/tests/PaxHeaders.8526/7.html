30 atime=1389151475.363878348
30 ctime=1389151475.363878348
