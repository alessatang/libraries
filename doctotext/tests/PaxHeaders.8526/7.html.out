30 mtime=1379553522.953587159
30 atime=1389151475.363878348
30 ctime=1389151475.363878348
